README.rst

```diff
@@ -851,6 +854,19 @@ function:
 **Note**: If there is already such a ``VERSION.lua`` file in the root of the
 application, it will be overwritten when packaged.
 
+If you would like to pass some parameters to unit files specify file name with
+parameters with flag ``--unit-params-file``. There is ``fd-limit`` option
+to set fd limit in ``LimitNOFILE`` parameter in ``systemd`` unit file and
+``systemd`` instantiated unit file. There is ``stateboard-fd-limit`` option
+to set fd limit in ``LimitNOFILE`` parameter in stateboard ``systemd`` unit file.
+
+.. code-block:: yaml
+
+    fd-limit: 1024
+    stateboard-fd-limit: 2048
+
+By default ``systemd-unit-params.yml`` from the project root is used as the file with parameters for unit files.
+
```

Comment:
Make it simpler:

You can specify some `systemd` units parameters using ``systemd-unit-params.yml`` file in the project root. It's possible to use another file specifying it in the ``--unit-params-file`` option.

These options are supported now:

* `fd-limit` - `LimitNOFILE` option for application instance;
* `stateboard-fd-limit` - `LimitNOFILE` option for stateboard instance.